src/arch/aarch64/include/asm/hal/reg.h

```diff
@@ -170,6 +179,18 @@ static inline uint64_t aarch64_ttbr0_el2_read(void) {
 	return read_system_reg(TTBR0_EL2);
 }
 
+static inline void aarch64_ttbr0_el2_write(uint64_t reg) {
+	write_system_reg(TTBR0_EL2, reg);
+}
+
+static inline uint64_t aarch64_ttbr0_el1_read(void) {
+	return read_system_reg(TTBR0_EL1);
+}
+
+static inline void aarch64_ttbr0_el1_write(uint64_t reg) {
+	write_system_reg(TTBR0_EL1, reg);
+}
+
 static inline uint64_t aarch64_ttbr1_el2_read(void) {
 	return 0;//read_system_reg(TTBR1_EL2);
```

Comment:
read_system_reg(TTBR1_EL2); brakes the system?

Reply:
Fixed.